.github/workflows/check-plugin-docs.js

```diff
@@ -1,21 +1,25 @@
 const fs = require("fs");
 const path = require("path");
-const fetchPluginDocs = require("../../website/components/remote-plugin-docs/utils/fetch-plugin-docs");
+const { resolvePluginDocs } = require("@hashicorp/platform-packer-plugins");
 
 const COLOR_RESET = "\x1b[0m";
 const COLOR_GREEN = "\x1b[32m";
 const COLOR_BLUE = "\x1b[34m";
 const COLOR_RED = "\x1b[31m";
 
+const VALID_PLUGIN_TIERS = ["official", "community", "verified"];
+
+
 async function checkPluginDocs() {
```

Comment:
Fixed this up, it was failing silently from what I could see, but should be running as expected going forward.